errors/daemon.go

```diff
@@ -966,4 +966,13 @@ var (
 		Description:    "Engine's predefined networks cannot be deleted",
 		HTTPStatusCode: http.StatusForbidden,
 	})
+
+	// ErrorCodeMultipleNetworkConnect is generated when more than one network is passed
+	// when creating a container
+	ErrorCodeMultipleNetworkConnect = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "CANT_CONNECT_TO_MULTIPLE_NETWORKS",
```

Comment:
There are other 7 instances of "CANT" which makes me now think that was a convention for this file ?! Also no agreement on `_` separated words.

`"CANTUNPAUSE","CANTSTOP", "CANTRESTART","EXECCANTRUN", "CANTDESTROY", "CANTSTART", "CANT_DELETE_PREDEFINED_NETWORK",`

I am not sure now if we should define and fix the error label convention as part of this PR.